suite-common/message-system/config/config.v1.json

```
@@ -877,6 +877,69 @@
                     }
                 }
             }
+        },
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": "<24.9.1",
+                        "mobile": "!",
+                        "web": "!"
```

Why not target outdated web clients too? Cached old version is not very likely, as the client would have to cache `html`, which browsers generaly never cache. But if it's a matter of changing one line, why not?
We do it [for a similar message](https://github.com/trezor/trezor-suite/blob/5c2cb7d048201ff7f72f3d7335fa180109b4f1ba/suite-common/message-system/config/config.v1.json#L743-L745).
```suggestion
                        "web": "<24.9.1"
```

---

6760d6c2abfe574314289160f84bae1e404b8f9b The message doesn't make much sense on web, but I guess it is better this way 👍